### ISSUE TITLE
Request for pretrained weights for Swin Transformer-Based Face Recognition (CASIA-WebFace + CosFace)

### ISSUE THREAD

**Issue by HectorPenades** [lxq1000/SwinFace#11]
Hello everyone,

In your article you speak about a model that was trained on the CASIA-WebFace dataset (494K images of 10,572 celebrities) using the CosFace loss function, which reportedly outperformed ArcFace for Swin Transformer-based models. Additionally, the FC-BN-FC-BN structure was mentioned to improve performance.

I was wondering if anyone could share this pretrained weights for this specific model (CASIA-WebFace + CosFace)? It would be really helpful for my.

Thank you in advance!

Best regards,
